### PR TITLE
Issue 38175: Correcting relative paths in extracted zip directories

### PR DESCRIPTION
**Pull request by labkey-ankurj**

### CODE REVIEW

File: internal/webapp/internal/ZipLoad/ZipLoad.js [hunk 1]
```diff
@@ -172,12 +172,24 @@ LABKEY.internal.ZipLoad = new function () {
         var filePath = file.fullPath.split('/');
         filePath.shift();
         var zipProgressName = filePath[filePath.length-1];
+        var fileZipPath = '';
+
+        for(var fp=0; fp<filePath.length; fp++) {
+            if(this.directoryBeingZipped === filePath[fp]) {
+                var index = fp;
+                while(index+1 < filePath.length) {
+                    fileZipPath += '/' + filePath[index+1];
```
Comment (labkey-jeckels) [inline internal/webapp/internal/ZipLoad/ZipLoad.js]: Can change this to filePath[++index] and remove the following line

File: internal/webapp/internal/ZipLoad/ZipLoad.js [hunk 2]
```diff
@@ -172,12 +172,24 @@ LABKEY.internal.ZipLoad = new function () {
         var filePath = file.fullPath.split('/');
         filePath.shift();
         var zipProgressName = filePath[filePath.length-1];
+        var fileZipPath = '';
```
Comment (labkey-jeckels) [inline internal/webapp/internal/ZipLoad/ZipLoad.js]: Please add logging and fallback to the file name if the directory never matches in the for() loop below.

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

File: internal/webapp/internal/ZipLoad/ZipLoad.js (hunk 1 above)
Comment (labkey-ankurj) [reply internal/webapp/internal/ZipLoad/ZipLoad.js]: Done.

File: internal/webapp/internal/ZipLoad/ZipLoad.js (hunk 2 above)
Comment (labkey-ankurj) [reply internal/webapp/internal/ZipLoad/ZipLoad.js]: Done

**Review by labkey-jeckels — APPROVED** [review]
Looks great, thanks.

**Comment by labkey-jony** [comment]
Client stated in ticket that the fix worked. They're expecting it to be rolled into 19.2

**Pull request closed without merging**